### PR TITLE
Dpdk primary secondary process

### PR DESCRIPTION
**Pull request by ader1990**
DPDK primary secondary test case required:
  * DPDK 19.05 or greater
  * Linux kernel 4.20 or greater
  * libibverbs version 23 or greater

Currently, libibverbs need to be compiled separately, as there is no support in Azure to install them directly.

### CODE REVIEW

File: Testscripts/Linux/build_install_rdma_core.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	exit 0
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+function build_install_rdma_core () {
+	SetTestStateRunning
+	packages=(git)
+	case "${DISTRO_NAME}" in
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/build_install_rdma_core.sh:23]: https://github.com/linux-rdma/rdma-core provides the each distro's installation steps.

**Comment by juhlee-microsoft** [comment]
Is it under PR test or still implementing? Why is it DO-NOT-MERGE label?

File: Testscripts/Linux/build_install_rdma_core.sh (hunk 1 above)
Comment (ader1990) [reply Testscripts/Linux/build_install_rdma_core.sh:23]: Currently, the rdma-core install/build is a hack to make the primary/secondary DPDK processes work on Azure, as the default version of the rdma-core package is lower than 23.

Once the rdma-core patches are backported or the version is changed in the supported distros, this part will be removed.

Note that this test case is supposed to work only with Linux kernel 4.20 or higher, DPDK 19.05 or higher, rdma-core 23 or higher.

The only Azure image that is close to these specifications is Ubuntu 19.04 (which has to have rdma-core upgraded, as the current default version is version 23).

Not even RHEL 8 has kernel > 4.20, it has 4.18.

**Review by adriansuhov — APPROVED** [review]

**Comment by ader1990** [comment]
Execution tested using kernel 4.20 and rdma-core stable-v24 :

```
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:26

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 DPDK                 VERIFY-DPDK-PRIMARY-SECONDARY-PROCESSES                                           PASS                16.85

```

Secondary process log:

```
EAL: Probing VFIO support...
EAL: WARNING: cpu flags constant_tsc=yes nonstop_tsc=no -> using unreliable clock cycles !
EAL: PCI device 86e8:00:02.0 on NUMA socket 0
EAL:   probe driver: 15b3:1004 net_mlx4
PMD: net_mlx4: PCI information matches, using device "mlx4_2" (VF: true)
PMD: net_mlx4: 1 port(s) detected
EAL: PCI device 92a2:00:02.0 on NUMA socket 0
EAL:   probe driver: 15b3:1004 net_mlx4
PMD: net_mlx4: PCI information matches, using device "mlx4_1" (VF: true)
PMD: net_mlx4: 1 port(s) detected
APP: Finished Process Init.
```

**Pull request merged**